hyperon_das/das.py

```diff
@@ -32,52 +28,14 @@ def __init__(self, **kwargs: Optional[Dict[str, Any]]) -> None:
         if atomdb_parameter == "ram":
             self.backend = InMemoryDB()
         elif atomdb_parameter == "redis_mongo":
-            mongo_hostname = kwargs.get('mongo_hostname')
-            mongo_port = kwargs.get('mongo_port')
-            mongo_username = kwargs.get('mongo_username')
-            mongo_password = kwargs.get('mongo_password')
-            mongo_tls_ca_file = kwargs.get('mongo_tls_ca_file')
-            redis_hostname = kwargs.get('redis_hostname')
-            redis_port = kwargs.get('redis_port')
-            redis_username = kwargs.get('redis_username')
-            redis_password = kwargs.get('redis_password')
-            redis_cluster = kwargs.get('redis_cluster')
-            redis_ssl = kwargs.get('redis_ssl')
-            required_parameters = [
-                mongo_hostname,
-                mongo_port,
-                mongo_username,
-                mongo_password,
-                redis_hostname,
-                redis_port,
-            ]
-            if not all(required_parameters):
-                raise InvalidDASParameters(
-                    message='Send required parameters to instantiate a RedisMongo',
-                    details="'mongo_hostname', 'mongo_port', 'mongo_username', 'mongo_password', 'redis_hostname', 'redis_port'",
-                )
-            self.backend = RedisMongoDB(
-                mongo_hostname=mongo_hostname,
-                mongo_port=mongo_port,
-                mongo_username=mongo_username,
-                mongo_password=mongo_password,
-                mongo_tls_ca_file=mongo_tls_ca_file,
-                redis_hostname=redis_hostname,
-                redis_port=redis_port,
-                redis_username=redis_username,
-                redis_password=redis_password,
-                redis_cluster=redis_cluster,
-                redis_ssl=redis_ssl,
-            )
+            self.backend = RedisMongoDB(**kwargs)
             if query_engine_parameter != "local":
-                raise InvalidDASParameters(
-                    message='Can`t instantiate a RedisMongo with query_engine=`local`'
-                )
+                raise InvalidDASParameters(message="query_engine parameter must be 'local'")
```

Comment:
Change messa to "'redis_mongo' backend requires local query engine ('query_engine=local')"

Reply:
ok